README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My complete name is *Miguel Ángel Rosales Legorreta*
+[My linkedin profile](www.linkedin.com)
+
+I'm doing my Master's degree in Public Policy at Harris
+
+Before comming to Chicago I worked in Mexico for
+* __The  Ministry of Finance__
```

Comment:
In this line, and in subsequent lines, you use two underlines `__` rather than two asterisks `**` to bold your text. This is a good use of alternative syntax, although it could become confusing if you are trying to use italics in Markdown, or want to combine syntax as Vivian suggested. For the future, you can either use the above syntax for bolded text, and either `_` or `*` for italics. Just be sure to keep track of what syntax you are using.

To Vivian's point, in order to combine both bold and italics, you can either type, `_**Ministry of Finance**_` or `*__Ministry of Finance__*` and they will both render as _**Ministry of Finance**_